x-pack/solutions/search/plugins/search_getting_started/public/components/footer/index.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiSpacer,
+  useCurrentEuiBreakpoint,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { DocCallouts } from './doc_callouts';
+import { SearchGettingStartedSectionHeading } from '../section_heading';
+import { footerLinks } from './footer_links';
+
+export const GettingStartedFooter = () => {
+  const currentBreakpoint = useCurrentEuiBreakpoint();
+  return (
+    <>
+      <EuiSpacer size="xxl" />
+      <EuiHorizontalRule />
+      <EuiSpacer size="xxl" />
+      <EuiFlexGroup direction={'column'} justifyContent="spaceBetween" gutterSize="l">
+        <SearchGettingStartedSectionHeading
+          title={i18n.translate('xpack.searchGettingStarted.footer.label', {
+            defaultMessage: 'Dive Deeper with Elasticsearch',
```

Comment:
```suggestion
            defaultMessage: 'Dive deeper with Elasticsearch',
```

Reply:
Thanks for the suggested change @florent-leborgne. I think I may have actually been looking at an old design, and it turns out the heading here isn't actually needed. I'm going to double check with @mdefazio, but I think I will need to remove this entirely.